vds_api_client/types.py

```diff
@@ -226,8 +226,8 @@ def filter(self, min_id=None, max_id=None,
     def __str__(self):
         if self.__bool__():
             nlen = max(max([len(r.name) for r in self._rois]), 10)  # max product length display
-            prod_ls = [(f' {roi.id:7d}  /  [{"X" if roi.display else " "}]  | {roi.name:{nlen}s} ' 
-                        f'| {roi.area:.3e} ha | {roi.created_at:%Y-%m-%d %H:%M} | {roi.description}')
```

Comment:
Indeed, the returned unit by the api was m2. Maybe we should think about returning units in the api for these kinds of variables

```json
{
  "area": {"unit": "m2", "value": 10202}
}
```